services/movie.py

```diff
@@ -0,0 +1,35 @@
+from db.models import Movie
+
+
+def get_movies(genres_ids: list[int] = None, actors_ids: list[int] = None):
```

Comment:
```suggestion
def get_movies(genres_ids: Optional[list[int]] = None, actors_ids: Optional[list[int]] = None):
```
Because None is default. Fix for all types